src/main/java/io/swagger/model/watchlist/WatchlistCreateRequest.java

```diff
@@ -0,0 +1,140 @@
+package io.swagger.model.watchlist;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.model.media.MediaItem;
+import io.swagger.v3.oas.annotations.media.Schema;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * WatchlistCreateRequest
+ */
+@Validated
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2022-04-02T22:43:09.213512-04:00[America/New_York]")
+
+
+public class WatchlistCreateRequest {
```

Comment:
`WatchlistCreateRequest` is a `Watchlist` but without the id. The schema validator was requiring an ID be passed on a POST request body (which I don't think should be required).